tests/unit/eWiseMul.cpp

```diff
@@ -139,6 +145,10 @@ void grb_program( const size_t &n, RC &rc ) {
 		rc = SUCCESS;
 	}
 	if( rc != SUCCESS ) { return; }
+	if( nonblocking_execution ) {
+		rc = rc ? rc : clear( out );
```

Comment:
The output error code of grb::clear is never expected. The spec says it should return SUCCESS; any other behaviour should probably cause this test to fail?